domain/application/config.go

```diff
@@ -11,3 +18,22 @@ const (
 	// StorageDefaultFilesystemSourceKey is the key for the default filesystem storage source.
 	StorageDefaultFilesystemSourceKey = "storage-default-filesystem-source"
 )
+
+// ConfigSchema returns the config schema and defaults for an application.
+func ApplicationConfigSchema() (configschema.Fields, schema.Defaults) {
+	return trustFields, trustDefaults
+}
+
+const defaultTrustLevel = false
+
+var trustFields = configschema.Fields{
+	application.TrustConfigOptionName: {
+		Description: "Does this application have access to trusted credentials",
+		Type:        configschema.Tbool,
+		Group:       configschema.JujuGroup,
+	},
+}
+
+var trustDefaults = schema.Defaults{
+	application.TrustConfigOptionName: defaultTrustLevel,
+}
```

Comment:
What are these for? They aren't used yet

Reply:
Ah, yes, dead code. I was doing it another way previously.